zest/releaser/utils.py

```diff
@@ -494,6 +518,47 @@ def system(command, input=''):
     return result
 
 
+def retry_command(command):
```

Comment:
I'd guess a different name would be better. `retry_command()` sounds like you're re-trying a previously issued command. Ideally, something like `system(cmd, allow_retry=True)`would be the clearest way.

Perhaps renaming `system()` would help? `execute_command()` and `execute_command_with_retry()`? Though I think `allow_retry` is clearest as an option.